.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+**Steps to reproduce**
+Give easy to follow steps that lead to the issue, for example:
+1. Execute `tns platform add android@next`
+2. Execute `tns run android`
+
```

Comment:
Can we add the requirement for the output of `tns doctor --log trace` . As the output is really long, we can set a requirement to place it in a gist.